src/GVF/Policies.jl

```diff
@@ -6,3 +8,9 @@ function get(π::AbstractPolicy, state_t, action_t, state_tp1, action_tp1, preds
 get(π::AbstractPolicy, state_t, action_t) = get(π::AbstractPolicy, state_t, action_t, nothing, nothing, nothing)
 
 
+struct NullPolicy <: AbstractPolicy
+end
+
+get(π::NullPolicy, state_t) = 1.0
```

Comment:
I'm not sure that I've seen this before. What is the use case for `NullPolicy`? It isn't a valid probability distribution, is it?